packages/supabase_flutter/README.md

```diff
@@ -124,15 +124,8 @@ Future<AuthResponse> signInWithGoogle() {
   /// You will have two different values for iOS and Android.
   const clientId = 'YOUR_CLIENT_ID_HERE';
 
-  /// TODO: Replace the following with your own app details
-  ///
-  /// Application ID or Bundle ID for your app.
-  /// If the application ID and Bundle ID is different for Android and iOS,
-  /// make sure the value here matches the platform you are running on.
-  const applicationId = 'com.supabase.example';
-
-  /// Fixed value for google login
-  const redirectUrl = '$applicationId:/google_auth';
+  /// reverse DNS form of the client ID + `:/` is set as the redirect URL
+  final redirectUrl = '${clientId.split('.').reversed.join('.')}:/';
```

Comment:
Is there no `google_auth` required at the end?

Reply:
Yeah, I misread it, but `google_auth` wasn't a requirement even for the first method either. 
https://stackoverflow.com/questions/30194644/how-to-handle-redirect-uri-mismatch-error-when-android-app-obtains-offline-acces